builtin/plugins/dposv3/storage.go

```diff
@@ -456,6 +474,13 @@ func GetReferrer(ctx contract.StaticContext, name string) *types.Address {
 }
 
 func SetReferrer(ctx contract.Context, name string, address *types.Address) error {
+	if ctx.FeatureEnabled(loomchain.DPOSVersion3_5, false) {
+		return ctx.Set(referrerKey(name), address)
+	}
+	return DeprecatedSetReferrer(ctx, name, address)
```

Comment:
Since the error from `SetReferrer` was ignored in `RegisterReferrer` we'll have to keep ignoring any error until the feature flag is enabled:
```golang
DeprecatedSetReferrer(ctx, name, address)
return nil
```